exchanges/wshandler/websocket_types.go

```diff
@@ -24,6 +24,9 @@ const (
 	WebsocketSubmitOrderSupported
 	WebsocketCancelOrderSupported
 	WebsocketWithdrawSupported
+	WebsocketMessageCorrelationSupported
+	WebsocketSequenceNumberSupported
+	WebsocketDeadMansSwitchSupported
```

Comment:
With the addtional functionality states being added is there ever a chance this grow over 32?
if so bit shifting will cause it to overflow 

could just drop the bit shift and have it set as iota

Reply:
I can see that happening eventually but right now we're at 17/32. I feel it should be addressed when the issue arises in a separate PR. @thrasher- your thoughts here regarding how we do websocket functionality? Are we moving away from bitshifting?

Reply:
How you've done it now is fine and can be addressed in a future PR incase we're at a risk of it overflowing